404.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
```

Comment:
Compare to https://github.com/allen-cell-animated/website-3d-cell-viewer/blob/main/public/gh-reroute/404.html